src/components/VmDetails/cards/SnapshotsCard/sagas.js

```diff
@@ -23,7 +23,7 @@ function* deleteVmSnapshot (action) {
   }
   yield put(addSnapshotRemovalPendingTask(snapshotId))
   let snapshotRemoved = false
-  for (let delaySec of [ 4, 4, 4, 30, 60 ]) {
+  for (let delaySec of [ 4, 4, 4, 30, 30, 60 ]) {
```

Comment:
You have something on the snapshot card that looks at the `pendingTasks` to try to show a snapshot as "delete in progress" correct?

Reply:
According to new design, it just hide `delete` and `restore` button for snapshot that is being deleted.